pkg/component/etcd/etcd/bootstrap.go

```diff
@@ -293,7 +294,7 @@ func (b *bootstrapper) Deploy(ctx context.Context) error {
 					{
 						Name:       webhookServerPortName,
 						Protocol:   corev1.ProtocolTCP,
-						Port:       webhookServerPort,
+						Port:       webhookServerServicePort,
```

Comment:
Any particular reason behind exposing the webhook server service port at `443` instead of just exposing `10250` in this case?

Reply:
No technical reason, just to be consistent with the setup of `gardener-resource-manager`.

Reply:
Thanks!